Sources/Target/Windows/Process.cpp

```diff
@@ -44,6 +44,12 @@ static uint8_t const gDebugBreakCode[] = {
 static const int gRtlExitUserThreadOffset = 0x02;
 #endif
 
+static DWORD getPageSize() {
```

Comment:
I think if you rebase on top of master you can get rid of this and just use `Platform::GetPageSize()`.